terra_utils/helpers/responses.py

```diff
@@ -0,0 +1,38 @@
+import io
+
+from django.core.files import File
+from django.http import HttpResponse, HttpResponseForbidden
+import magic
+
+from .. import settings
+
+
+def get_media_response(request, data, permissions=None, headers=None):
+    # For compatibility purpose
+    content, url = None, None
+    if isinstance(data, (io.IOBase, File)):
+        content, url = data, data.url
```

Comment:
I think that io.IOBase has not url attribute, it is specific to django file

Reply:
You're right, fixed